src/main/resources/com/puppycrawl/tools/checkstyle/grammar/java.g

```diff
@@ -1166,16 +1181,11 @@ traditionalStatement
         // Return an expression
         |    "return"^ (expression)? SEMI
 
-        // switch/case statement
-        |    "switch"^ LPAREN expression RPAREN LCURLY
-                ( casesGroup )*
-            RCURLY
-
```

Comment:
if movement to line 1139 was not done on good reason, please do this update in separate commit.

Reply:
Fixed.